.nvmrc

```diff
@@ -0,0 +1 @@
+v16.0.0
```

Comment:
We are using [asdf & nix](https://github.com/smartcontractkit/chainlink-terra/blob/351e251ba4e76e2016ab93cbb1730fc89d94a55b/Makefile#L24) so adding a third version manager is probably unnecessary.

Please remove this file and use/tweak one of the above.